test/html/multiple-imports.html

```diff
@@ -7,5 +7,6 @@
     Code distributed by Google as part of the polymer project is also
     subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+<link rel="import" href="multiple-imports.html">
```

Comment:
Okay I force pushed a new version of the branch with a test that specifically addresses sharded build strategy (which fails) and subsequent commit contains the fix code (same fix code).